pkg/policy/mapstate.go

```diff
@@ -104,6 +110,7 @@ type Key struct {
 func (k Key) String() string {
 	return "Identity=" + strconv.FormatUint(uint64(k.Identity), 10) +
 		",DestPort=" + strconv.FormatUint(uint64(k.DestPort), 10) +
+		",PortMask=" + strconv.FormatUint(uint64(k.PortMask), 10) +
 		",Nexthdr=" + strconv.FormatUint(uint64(k.Nexthdr), 10) +
 		",TrafficDirection=" + strconv.FormatUint(uint64(k.TrafficDirection), 10)
```

Comment:
I would suggest to append another value to the string that contains the "calculated end port".